src/status_im2/navigation/roots.cljs

```diff
@@ -148,10 +139,19 @@
   []
   ;;TABS
   (merge (old-roots)
+         ;;INTRO (onboarding carousel)
+         {:intro-stack
+          {:root {:stack {:id       :intro
+                          :children [{:component {:name :intro
+                                                  :id :intro
+                                                  :options
+                                                  {:statusBar {:style :light}
```

Comment:
we can also use `(merge (status-bar-options))` here

Reply:
yeah sure, it's just that this needs to override with light here.

Reply:
wouldn't that will be handled by `status-bar-options` function itself - https://github.com/status-im/status-mobile/pull/15127#discussion_r1123479978

Reply:
I cleaned this up a small bit it's needs :style :light here for this to look correct on this screen.